superset/charts/api.py

```diff
@@ -457,7 +432,11 @@ def data(self) -> Response:
         if not request.is_json:
```

Comment:
Suggestion, would be great to define the response schema has a marshmallow schema also and add it to the common schema OpenAPI spec, this would reduce the yaml doc even further

Reply:
Agreed; I was so focused on the request that I completely overlooked the response.